spec/test_spec.rb

```diff
@@ -1 +1,94 @@
+require 'fileutils'
+require 'pry'
```

Comment:
remove this before merging to master(since this is only for debug purposes)

Reply:
done